seeds/_sector/dex/baso_finance_base_base_trades_seed.csv

```diff
@@ -0,0 +1,4 @@
+blockchain,project,version,block_date,tx_hash,evt_index,token_bought_address,token_sold_address,block_number,token_bought_amount_raw,token_sold_amount_raw
+base,baso_finance,1,2024-02-29,0xffd4b1a0feccb552677f66dfad9c9513ab099e8861df3c9680a71403fbdc01a8,97,0xd9aaec86b65d86f6a7b5b1b0c42ffa531710b6ca,0x4200000000000000000000000000000000000006,11219567,20130916,5917311547268586
+base,baso_finance,1,2024-02-29,0x77b30ef875f23152d952956180bb110cf87c583c24d8922e605630503c8a9fa0,8,0x4200000000000000000000000000000000000006,0xd9aaec86b65d86f6a7b5b1b0c42ffa531710b6ca,11218505,3002100810131972,10328676
```

Comment:
The version you record here is `1`, while in `baso_finance_base.base_trades` is `2`.
https://github.com/duneanalytics/spellbook/pull/5476/files#diff-df7fe6ef229169722ae0a95c5982d333b6a90199dd992bc5a832131a9993d2b1R17